client/app/host/TeamInspector.tsx

```diff
@@ -0,0 +1,116 @@
+                                )}
+                                {team.name}
+                            </span>
+                        </SelectItem>
+                    ))}
+                </SelectContent>
+            </Select>
+            <Separator className="my-2" />
+            {selectedTeam && (
+                <Card className="flex h-full w-full flex-col">
+                    <CardHeader className="w-full">
+                        <CardTitle className="flex items-center justify-between text-2xl">
+                            {selectedTeam.name}
+                            {selectedTeam.status ? (
+                                <p className="text-green-500">Connected</p>
+                            ) : (
+                                <p className="text-gray-300 dark:text-gray-500">Disconnected</p>
+                            )}
+                        </CardTitle>
+                    </CardHeader>
+                    <CardContent className="flex h-full w-full flex-col gap-4 text-lg">
+                        <span className="flex justify-between align-middle">
+                            <p>
+                                <strong>Points: </strong>
+                                {selectedTeam.points}
+                            </p>
+                            <Button
+                                variant="outline"
+                                onClick={() => {
+                                    toast({
+                                        title: 'Coming Soon',
+                                        description: 'This feature is coming soon!',
+                                        variant: 'destructive',
+                                    });
+                                }}
+                            >
+                                Submission History
+                            </Button>
+                        </span>
+                        <div className="flex h-full flex-col rounded border p-4">
+                            <span className="mt-auto flex w-auto gap-1 align-middle">
+                                <Input type="text" placeholder="Message..." />
+                                <Button
+                                    variant="secondary"
+                                    onClick={() => {
+                                        toast({
+                                            title: 'Coming Soon',
+                                            description: 'This feature is coming soon!',
+                                            variant: 'destructive',
+                                        });
+                                    }}
+                                >
+                                    <Send />
+                                </Button>
+                            </span>
+                        </div>
+                    </CardContent>
+                </Card>
```

Comment:
I'm not a big fan of this being a card- it feels like a lot of nesting.  It may look nicer to have it just be in the body directly.  There is already the separator at the top.